### PR TITLE
Added Hautelook AliceBundle v2

### PR DESCRIPTION
**Pull request by jmsche**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: hautelook/alice-bundle/2.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Nelmio\\Alice\\Bridge\\Symfony\\NelmioAliceBundle": ["dev", "test"],
```
Comment (Nyholm) [inline hautelook/alice-bundle/2.0/manifest.json]: This should be a separate recipe

File: hautelook/alice-bundle/2.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        "Fidry\\AliceDataFixtures\\Bridge\\Symfony\\FidryAliceDataFixturesBundle": ["dev", "test"],
```
Comment (Nyholm) [inline hautelook/alice-bundle/2.0/manifest.json]: This should be a separate recipe

**Comment by Nyholm** [comment]
Any updates on this PR?

**Comment by theofidry** [comment]
This PR requires #258

**Comment by Nyholm** [comment]
This PR is no longer blocked. Let me know when you are ready for another review

File: hautelook/alice-bundle/2.0/manifest.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Nelmio\\Alice\\Bridge\\Symfony\\NelmioAliceBundle": ["dev", "test"],
```
Comment (theofidry) [inline hautelook/alice-bundle/2.0/manifest.json]: I think this one should be removed as well :)

Comment (jmsche) [reply hautelook/alice-bundle/2.0/manifest.json]: My bad, didn't see the nelmio/alice recipe ;)

**Comment by renatomefi** [comment]
@symfony-flex-server[bot] redeploy please?

**Comment by renatomefi** [comment]
@symfony-bot redeploy?

**Comment by jmsche** [comment]
@symfony-flex-server review please

**Comment by jmsche** [comment]
@Nyholm The PR has been fixed to match your requested changes.

@theofidry I guess the PR will never pass if the DoctrineBundle is not required in AliceBundle, or if this check is not removed: https://github.com/hautelook/AliceBundle/blob/master/src/DependencyInjection/HautelookAliceExtension.php#L48

**Comment by theofidry** [comment]
@jmsche indeed looks like https://github.com/hautelook/AliceBundle/pull/391 is a blocker

**Pull request closed without merging**